src/profile-logic/marker-data.js

```diff
@@ -473,9 +473,14 @@ export function mergeStartAndEndNetworkMarker(markers: Marker[]): Marker[] {
             marker.data.status === 'STATUS_START'
               ? [marker, markerNext]
               : [markerNext, marker];
+          const startData: NetworkPayload = (startMarker.data: any);
+          const endData: NetworkPayload = (endMarker.data: any);
```

Comment:
This isn't pretty but Flow doesn't know that it's correct. I plan to rework this by moving all this code to `deriveMarkersFroMRawMarkerTable` so I didn't want to spend too much time on this.